src/SuperDumpService/Program.cs

```diff
@@ -5,7 +5,7 @@ namespace SuperDumpService {
 	public static class Program {
 		public static void Main(string[] args) {
 			var host = new WebHostBuilder()
-				.UseKestrel()
+				.UseKestrel(opt => opt.Limits.MaxRequestBodySize = 8589934592L)  // 8gb
```

Comment:
hmmm... too bad. it's kind of a mess, to have the same setting in two places and one of them is hard-coded.

just found a sample where ConfigurationBuilder is instantiated in Main(): https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration. Maybe we can build configuration soon enough?